src/builds/travis.rs

```diff
@@ -0,0 +1,97 @@
+use std::io::Read;
+
+use chrono::{DateTime, UTC};
+use diesel;
+use diesel::prelude::*;
+use hyper::Client;
+use hyper::header::UserAgent;
+use hyper::net::HttpsConnector;
+use hyper_native_tls::NativeTlsClient;
+use serde::de::DeserializeOwned;
+use serde_json;
+
+use DB_POOL;
+use domain::builds::Build;
+use error::DashResult;
+
+header! { (Accept, "Accept") => [String] }
+
+const ROOT_URL: &'static str = "https://api.travis-ci.org";
+const UA: &'static str = "rusty-dash/0.0.0";
```

Comment:
Ah, I see! We should only ever be running one version of this, but it might be useful to set it to something like `concat!("rusty-dash/", env!("CARGO_PKG_VERSION"))` just in case.